.travis.yml

```diff
@@ -11,6 +11,7 @@ python:
   - 3.6
   - 3.7
   - 3.8
+  - 3.9-dev
```

Comment:
I’m actually wondering if we could do without this until 3.9 reaches beta? With 3mo until the final release we’ll have lots of time to report back any issues. At this point we’re trying to keep maintenance overhead as low as possible in this project. (We could add 3.9 to allow_failures but it would defeat the point as builds wouldn’t report as errors...)

Reply:
Sure, it can wait. At least it's done one run now and nothing untoward happened. Feel free to close this.

Reply:
Yes, it’s good to know that we’re not currently broken. :) Thank you!